microprofile/WFLY-12737_quickstart-for-mp-health-2.1.adoc

```diff
@@ -0,0 +1,65 @@
+= WFLY-12737 Quickstart for MP Health 2.1
+:author:            Martin Stefanko
+:email:             mstefank@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This RFE provides a new quickstart and guide for MicroProfile Health 2.1 use in WildFly /
+JBoss EAP. The example will cover all basic functionalities of MicroProfile Health 2.1,
+namely: liveness and readiness checks, the basic usage of the `HealthCheckResponseBuilder`,
+and the inclusion of user specific data in the health check responses.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-12737[WFLY-12737]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1368[EAP7-1368]
+
+=== Dev Contacts
+
+* mailto:mstefank@redhat.com[Martin Stefanko]
+
+=== QE Contacts
+
+* TBD
```

Comment:
Despite the fact this RFE is tested by engineering, feel free to write me as a QE contact

Reply:
@zhantemirov updated. Can you take a look again, please?